src/utils/db.js

```diff
@@ -0,0 +1 @@
+export const db = "http://localhost:4000";
```

Comment:
I think it will offer greater flexibility if we read the backend url from the environment.